sdk/core/core-client/review/core-client.api.md

```diff
@@ -77,12 +76,6 @@ export function createSerializer(modelMappers?: {
     [key: string]: any;
 }, isXML?: boolean): Serializer;
 
-// @public
-export function createSpanFunction({ packagePrefix, namespace }: SpanConfig): <T extends OperationOptions>(operationName: string, operationOptions: T) => {
```

Comment:
It would also be a good idea to add a breaking change note in the changelog along with this change

Reply:
Ah, great point, will do!

Reply:
Done!